digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/PedestrianCrossingService.scala

```diff
@@ -39,5 +40,88 @@ class PedestrianCrossingService(val roadLinkService: RoadLinkService) extends Po
     }
     id
   }
+
+  override def getByBoundingBox(user: User, bounds: BoundingRectangle): Seq[PersistedAsset] = {
```

Comment:
It seems like there is a lot of duplicated code we can have a method on the base class that has a function as a parameter, and then reuse it.